src/pudl/analysis/service_territory.py

```diff
@@ -30,8 +30,6 @@
 MAP_CRS = "EPSG:3857"  # For mapping w/ OSM baselayer tiles
 CALC_CRS = "ESRI:102003"  # For accurate area calculations
 
-ENTITY_TYPE = {"ba": "balancing_authorities", "util": "utilities"}
-
```

Comment:
It was simpler to just avoid having this mapping. Maybe I should have made an `Enum`?